src/main/java/teammates/ui/controller/StudentCourseJoinAuthenticatedAction.java

```diff
@@ -24,8 +25,16 @@
     @Override
     protected ActionResult execute() throws EntityDoesNotExistException {
         Assumption.assertNotNull(regkey);
```

Comment:
Will be hard to do that, because the `regkey` param comes from the base `Action` class. Also, really it shouldn't be the case that regkey contains something else, if not for this `${amp}` workaround. I think the best I can do is adding some comments there.